examples/with-tailwind/packages/ui/tsup.config.ts

```diff
@@ -8,6 +8,6 @@ export default defineConfig((options: Options) => ({
   dts: true,
   minify: true,
   clean: true,
-  external: ["react", "react-dom"],
```

Comment:
This enumeration was unnecessary and should have instead been populated by tools. (Human error is expected, this is the kind of thing that should have been automatic.)